servers/quarkus-server/src/main/java/org/projectnessie/server/providers/DynamoVersionStoreFactory.java

```diff
@@ -34,31 +35,31 @@
 @Dependent
 public class DynamoVersionStoreFactory implements VersionStoreFactory {
   private final DynamoDbClient dynamoDbClient;
+  private final QuarkusDatabaseAdapterConfig config;
 
   /** Creates a factory for dynamodb version stores. */
   @Inject
-  public DynamoVersionStoreFactory(DynamoDbClient dynamoDbClient) {
+  public DynamoVersionStoreFactory(
+      DynamoDbClient dynamoDbClient, QuarkusDatabaseAdapterConfig config) {
     this.dynamoDbClient = dynamoDbClient;
+    this.config = config;
   }
 
   @Override
   public <VALUE, METADATA, VALUE_TYPE extends Enum<VALUE_TYPE>>
       VersionStore<VALUE, METADATA, VALUE_TYPE> newStore(
           StoreWorker<VALUE, METADATA, VALUE_TYPE> worker, ServerConfig serverConfig) {
 
+    DynamoDatabaseClient client = new DynamoDatabaseClient();
+    client.configure(
+        ImmutableProvidedDynamoClientConfig.builder().dynamoDbClient(dynamoDbClient).build());
```

Comment:
Can you move the verbose `Immutable...` code into a static method `ProvidedDynamoClientConfig.of(DynamoDbClient dynamoDbClient)` (and similar for the other version store types)?

Reply:
Fixed in 90b8280422f3dab555be727df30151ec99be8856